File: Lovecraft/Api/Model/Event.cs
namespace Lovecraft.Api.Model;

public class Event
{
	public int Id { get; set; }
	public string Title { get; set; }
	public string Rules { get; set; }
	public string CoverUrl { get; set; }

	public DateTime DateBegin { get; set; }
	public DateTime DateEnd { get; set; }

	public virtual List<Story> stories { get; set; }

}

File: lovecraft-ts/src/schemas/Event.ts
import { Prisma } from '@prisma/client'

export interface IEventDto {
  id:               number
  title:            string
  rules:            string
  coverUrl:         string
  dateBegin:        Date
  dateEnd:          Date
  hasAlreadyParticipate: Boolean
  createdAt:        Date
  updatedAt:        Date
}

export interface IPostEventDto {
  title:            string
  rules:            string
  coverUrl:         string
  dateBegin:        Date
  dateEnd:          Date
}

export type EventWithRelations = Prisma.EventGetPayload<{
  include: {
    stories: true
  }
}>